extensions/amp-accordion/1.0/accordion.js

```diff
@@ -88,19 +92,31 @@ function AccordionWithRef(
           expandSingleSection
         );
       });
-      return () => setExpandedMap((expandedMap) => omit(expandedMap, id));
+      eventMap.current[id] = {onExpand, onCollapse};
+      return () => {
+        setExpandedMap((expandedMap) => omit(expandedMap, id));
+        omit(/** @type {!Object} */ (eventMap.current), id);
+      };
     },
     [expandSingleSection]
   );
 
   const toggleExpanded = useCallback(
     (id) => {
+      const newValue = !expandedMap[id];
```

Comment:
Thanks, moved the `newValue` inside of the `setExpandedMap` callback.  However, things break if I trigger the event within the callback, so I kept the event triggering outside of the callback.  I declare the `newValue` outside of the callback then initialize it within the callback so it can be used there.  Then also use it outside the callback since its value is initialized.

See latest.  Let me know what you think!